sunpy/tests/helpers.py

```diff
@@ -22,7 +22,7 @@
     SKIP_GLYMUR = True
 else:
     # See if we have a C backend
-    if any((glymur.lib.openjp2.OPENJP2, glymur.lib.openjpeg.OPENJPEG)):
+    if glymur.lib.openjp2.OPENJP2:
```

Comment:
Unless we want to support testing with glymur<0.9 when openjp2 doesn't work but openjpeg does, this change should fix things.

Reply:
Does this chance actually mean we have a min version of 0.9.1 for sunpy now?

Reply:
Or we have a min version of openjpeg of 0.2.3, which was released 4th Oct 2017? (http://www.openjpeg.org/)

Reply:
I'm not sure how we enforce that, which we can do with python packages?

Reply:
Sorry I got a bit confused, the requirement I suggested above is only for running the tests. Users should be able to go on using openjpeg<2 with glymur<0.9 if they want.

Alternatively we could add some fancier conditional logic here based on `LooseVersion(glymur)` to handle <0.9 and >0.9.

Reply:
I also got confused, I didn't notice this was a test file.

Reply:
This is to work out if we should skip the openjpeg tests if glymur is installed but openjpeg isn't